src/app/modules/item/components/item-chapter-view/group-situation-chapter-view/user-progress/user-progress.component.ts

```diff
@@ -20,11 +19,21 @@ export class UserProgressComponent implements OnChanges {
 
   state: 'success'|'in-progress'|'no-score'|'not-started' = 'no-score';
 
+  @ViewChild(PermissionsEditDialogComponent) dialog!: PermissionsEditDialogComponent;
```

Comment:
I don't see why you would need to store your child more than at another place. You can pass arg without it. And you can use an input-output if you need to share "visible" (but I don't even see why it would be useful).

Reply:
fixed, I use a simple Input now